api/tests/phones_views_tests.py

```diff
@@ -1280,6 +1280,8 @@ def multi_reply(
         ("+13025550001", "text"),  # Same last 4 digits as below
         ("+13035550001", "text"),  # Same last 4 digits as above
         ("+13045551301", "text"),  # Last 4 match first 4 of oldest
+        ("64192", "text"),  # shortcode
```

Comment:
✅ I checked that this test fails with `phonenumbers.phonenumberutil.NumberParseException: (0) Missing or invalid default region.` without the fix.